runtime/parachains/src/scheduler/tests.rs

```diff
@@ -965,27 +964,17 @@ fn availability_predicate_works() {
 		..Default::default()
 	};
 
-	let HostConfiguration {
-		group_rotation_frequency,
-		chain_availability_period,
-		thread_availability_period,
-		..
-	} = default_config();
-	let collator = CollatorId::from(Sr25519Keyring::Alice.public());
+	let HostConfiguration { group_rotation_frequency, paras_availability_period, .. } =
+		default_config();
 
-	assert!(
-		chain_availability_period < thread_availability_period &&
-			thread_availability_period < group_rotation_frequency
-	);
+	assert!(paras_availability_period < group_rotation_frequency);
```

Comment:
Not sure what this assert was about, but it is very likely no longer relevant.

Reply:
I guess that assert was about making sure a specific ordering that the test assumes was satisfied.